ui/src/components/Header/index.tsx

```diff
@@ -1,61 +1,66 @@
-import AppBar from "@mui/material/AppBar"
-import { Box, Button, Container, Toolbar, Typography } from "@mui/material"
-import { Link } from "react-router-dom"
-import MultiProxySelection from "../MultiProxySelection"
-import { useAccounts } from "../../contexts/AccountsContext"
-import styled from "styled-components";
+import {Toolbar, Typography} from "@mui/material"
+import styled from "styled-components"
+import {styled as styledMaterial} from '@mui/material/styles';
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import * as React from "react";
+import MuiAppBar, {AppBarProps as MuiAppBarProps} from '@mui/material/AppBar';
+import MultiProxySelection from "../MultiProxySelection";
 // import NetworkSelection from "../NetworkSelection"
 
 interface Props {
+  drawerWidth: number,
   className?: string
+  handleDrawerOpen: () => void,
+  open: boolean
 }
+interface AppBarProps extends MuiAppBarProps {
+  open?: boolean;
+  drawerWidth: number
 
-const Header = ({ className }: Props) => {
-  const { accountList } = useAccounts()
+}
+
+const AppBar = styledMaterial(MuiAppBar, {
+  shouldForwardProp: (prop) => prop !== 'open'
+})<AppBarProps>(({ theme, open, drawerWidth }) => ({
+  transition: theme.transitions.create(['margin', 'width'], {
```

Comment:
It's magic to me, could we use good old css if that's possible? it's easier to understand and maintain.